bare-metal/elastic-metal/reference-content/elastic-metal-rv1-advanced-usage.mdx

```diff
@@ -0,0 +1,106 @@
+---
+meta:
+  title: Elastic Metal RV1 advanced usage
+  description: This page explains some advanced ways to use EM-RV1 servers.
+content:
+  h1: Elastic Metal RV1 advanced usage
+  paragraph: This page explains some advanced ways to use EM-RV1 servers.
+tags: elastic-metal server riscv em-rv1
+dates:
+  validation: 2024-03-15
+  posted: 2024-03-15
+categories:
+  - bare-metal
+---
+
+## Boot process
+
+EM-RV1 servers do not support standard UEFI or BIOS boot, therefore the boot
+process might slightly differ from what you are used to on other servers. At
+boot, the bootloader expects the eMMC to be partitioned as GPT, and will look
+for a `boot.itb` file in the first or second partition. The partition that
+contains this boot file must be formatted as EXT4.
+
+This `boot.itb` file is in fact a [FIT
+Image](https://docs.u-boot.org/en/latest/usage/fit/source_file_format.html)
+that must contain the following sections:
+
+  - **kernel**: A Linux kernel image.
+  - **fdt**: A [device tree](https://en.wikipedia.org/wiki/Devicetree).
+  - **opensbi**: The SBI as defined by [RISC-V SBI
+      specification](https://github.com/riscv-non-isa/riscv-sbi-doc/).
+  - **env**: A text-based environment file that defines the following keys:
+      - `usr_bootargs`: Command-line arguments to pass to the kernel.
+      - `usr_has_ramdisk`: Set to `1` if we should load the ramdisk from the FIT
+        image.
+  - **ramdisk** (optional): A ramdisk image.
+
+The following section will explain in details how to create your own boot FIT image.
+
+## Boot a custom kernel
+
+In this section, you will build a boot image based off a close-to-mainline
+Linux kernel.
+
+<Message type="note">
+  Mainline Linux is not fully compatible with EM-RV1 servers yet.
+</Message>
+
+<Macro id="requirements" />
+<Message type="requirement">
```

Comment:
Is this redundant ?

Reply:
I actually don't know, but this is the way it's done in many places:

https://github.com/scaleway/docs-content/blob/ab4ae50353a8a935d32c875f0d134c40d970ea44/compute/instances/how-to/manage-volumes.mdx#L49-L54

I can't check if it renders differently because building just this page already eats up all my RAM.

Reply:
Got it. It was indeed redundant.